acceptance/tasks/acceptance.rake

```diff
@@ -0,0 +1,56 @@
+require 'rake'
+
+def generate_hosts(pe_version, tests)
```

Comment:
The tests parameter isn't used here.

Reply:
This should probably also take the test target and hypervisor as parameters. We want to use the abs hypervisor for our CI jobs (that's the CI scheduler), but also want the option to use vmpooler for local runs. Might want to add an option to specify the hypervisor in the task itself. Think it should be required so people are forced to specify it but I don't have a strong preference.

Reply:
I'll add the arguments to the method, also not sure if it'll be worthwhile to specify the hypervisor in the task. I'll omit that for now for the sake of simplicity, it can be added pretty easily later. I generally prefer the argument-with-defaults pattern, so regardless I do not think it should be required.